### ISSUE TITLE
add feature `inherit_auto_import`

### ISSUE THREAD

**Issue by drewmullen** [aws-ia/terraform-aws-ipam#16]
to support DRY: add a flag to allow sub_pools to inherit parent `auto_import`

**Comment by nitrocode** [comment]
Isn't that [top_auto_import](https://github.com/aws-ia/terraform-aws-ipam#input_top_auto_import) ?

**Comment by drewmullen** [comment]
No, that just sets `auto_import` at the top level pool. currently you have to set `auto_import` at every level. pools, by design dont inherit any attributes from the parent pools
